docs/user/setting_up_your_development_environment.md

```diff
@@ -0,0 +1,101 @@
+# Setting up your development environment
```

Comment:
perhaps you walk the user through a quick check to make sure everything is working.

possibilities:
* they can run `make docs` to generate the documentation
* they can run `pytest` on some smallish test to make sure they can run tests
* they can run `make test` to verify that Docker is working (though this requires they install docker -- you should note that since it's the primary way of running tests outside of the CI)
* they can edit some file in the repo which prints something on import/construction, and verify that when they run a launcher it prints (as opposed to not printing, which would suggest they have not successfully installed the package as editable).